deployer/src/deployer/analyze_pr.py

```diff
@@ -99,7 +99,7 @@ def post_about_deployment(build_directory: Path, **config):
 
 
 def mdn_url_to_dev_url(prefix, mdn_url):
-    template = "https://{prefix}.content.dev.mdn.mozit.cloud{mdn_url}"
+    template = "[{mdn_url}](https://{prefix}.content.dev.mdn.mozit.cloud{mdn_url})"
```

Comment:
According to the name of the function, it returns a url, but now it returns a link. Let's pull this up, keeping this function as is.

```suggestion
    template = "https://{prefix}.content.dev.mdn.mozit.cloud{mdn_url}"
```